Tests/AppStoreConnectClientTests/TestHelpers/MockObjects.swift

```diff
@@ -123,4 +148,50 @@ enum MockObjects {
         )
         return response
     }
+    
+    static var prereleaseVersion: Components.Schemas.PrereleaseVersionWithoutIncludesResponse {
+        let dateFormatter = DateFormatter()
+        let mockData = dateFormatter.date(from: "2011-11-11 11:11:11 +0000")
+        let response = Components.Schemas.PrereleaseVersionWithoutIncludesResponse(
+            data: .init(
+                _type: .preReleaseVersions,
+                id: "FooBarId",
+                attributes: .init(
+                    version: "Foo",
+                    platform: .IOS,
+                    uploadedDate: mockData,
+                    expirationDate: mockData,
+                    expired: true,
+                    minOsVersion: "FooMinOs",
+                    lsMinimumSystemVersion: "BarMinSysVer",
+                    computedMinMacOsVersion: "BazCompMinMacOsVer",
+                    iconAssetToken: .none,
+                    processingState: .PROCESSING,
+                    buildAudienceType: .APP_STORE_ELIGIBLE,
+                    usesNonExemptEncryption: false
+                ),
+                relationships: .none,
+                links: .init(_self: "FooBarBazLink1")
+            ),
+            links: .init(_self: "FooBarBazLink2")
+        )
+        return response
+    }
+    
+    static var buildSchema: Components.Schemas.Build {
```

Comment:
What is the difference between this var and the one on line 90? 
Perhaps we should leave one. 
Or give them names so that they are clearer than those that are used.

Reply:
buildSchema removed.